examples/machining.yaml

```diff
@@ -0,0 +1,430 @@
+# Example: Machine shop, one step of threading a part
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - alice: https://alice.example/
+    bob: https://bob.example/
+    machine: https://machine-shop.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/machining.yaml
+'@graph':
+
+  # recipe
+
+  - '@id': machine:a7e94dde-2997-451e-a7e1-8d364d5e6dee
+    '@type': ResourceSpecification
+    name: Threaded part 87
+    unit: unit:Number
+    substitutable: True
+
+  - '@id': machine:2d350055-acba-4e17-92f8-8b126272ee40
+    '@type': ResourceSpecification
+    name: Unthreaded part 45
+    unit: unit:Number
+    substitutable: True
+
+  - '@id': machine:07819789-dd51-44c3-b35c-9210165bc832
+    '@type': ResourceSpecification
+    classifiedAs: https://www.wikidata.org/wiki/Q2211295 # drilling machine
+    name: Drill Press
+    unit: unit:Number
+    unitOfUse: unit:Hour # want a better way to do this!
```

Comment:
Would you ever want to put not a time unit here? Also what 'unit' means here, can one use machine for fractions of unit, for example .5 hour?

Reply:
Yes, fractions would be perfectly valid.  An example where unit of use may not be a time unit is printing press impressions.

Reply:
Ok, and vehicle could have 'distance', still for scheduling usage will need to have duration.